src/bin/bobp.rs

```diff
@@ -318,6 +320,10 @@ fn stat_worker(
     );
 }
 
+fn average(values: &[f64]) -> f64 {
+    finite_or_default(values.iter().sum::<f64>() / values.len() as f64)
```

Comment:
better to have average in a window. @pyakushin

Reply:
It can be both. Average on all values for results and moving average for current statistics

Reply:
It is final stats, so I think it is fine